src/client/package.json

```diff
@@ -20,7 +20,7 @@
   },
   "scripts": {
     "precommit": "lint-staged",
-    "start": "cross-env REACT_APP_BUILD_VERSION=$(git describe) react-scripts start",
```

Comment:
@rikoe I take it we're happy for these other changes to find their way into this release branch?

Reply:
yeah that's fine